packages/plugin/src/types.ts

```diff
@@ -76,6 +77,7 @@ export interface VersionMetadata {
 	versionPath: string; // /baseUrl/api/1.0.0
 	versionBadge: boolean;
 	versionBanner: VersionBanner | null;
+	versionGitRefName: string; // master
```

Comment:
This needs to get populated somewhere

Reply:
It looks like version metadata is quite Docusaurus Docs-specific, so I'll leave it out for now since that could be a separate PR.